README.md

```diff
@@ -40,11 +40,11 @@ This version is out of date. The latest official release is 1.7.0
 Success! `julia-1.7.0-rc3` now to point to 1.7.0-rc3
 v"1.7.0-rc3"
 
-julia> update_julia("1.8") # nightly, update_julia("nightly") also works
-installing julia 1.8.0-DEV
+julia> update_julia("1.10") # nightly, update_julia("nightly") also works
+installing julia 1.10.0-DEV
 This version is an experimental development build not recommended for most users. The latest
-official release is 1.7.0
-Success! `julia-1.8` & `julia-1.8.0-DEV` now to point to 1.8.0-DEV
+official release is 1.9.3
+Success! `julia-1.10` & `julia-1.10.0-DEV` now to point to 1.10.0-DEV
```

Comment:
There's no need to update these versions. And if we do, it should be from rerunning the commands in a repl and copy-and-pasting the output to make sure we're not internally inconsistent (in this case the return value does not align with the function argument after your change)